packages/core-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terra-ui-packages/core-utils",
-  "version": "0.0.6",
+  "version": "0.1.6",
```

Comment:
If we're going to bump this version, let's go to either `0.0.7` or `0.1.0`.

Reply:
major = breaking change
minor = additive (non-breaking) change
patch = bugfix...

Since PR is adding stuff, minor version bump...

Reply:
Yes, but there's no need to go straight to 0.1.6. Bumping one version part can reset the following parts to zero.

Reply:
According to the SemVer spec, a minor version increase _must_ reset the patch version to 0.
https://semver.org/#spec-item-7

Reply:
ok fixed, and thx for checking the homework :)